### ISSUE TITLE
AoT builds failing

### ISSUE THREAD

**Issue by spiffyjr** [swimlane/ngx-datatable#667]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
AoT builds fails with `ngx-datatable/release/components/body/body-cell.component.ngfactory.ts (1,1): Property 'checkboxable' does not exist on type 'TableColumn'.`.

**Expected behavior**
AoT build succeeds.

**Reproduction of the problem**
Build a project using AoT.

**Comment by spiffyjr** [comment]
Actually, this might be fixed in 1a37a732a5537341492c4609707f083d808a88d9. Checking now.

**Comment by spiffyjr** [comment]
Confirmed as fixed. Closing.
